Routing/RestRouteCollection.php

```diff
@@ -76,10 +76,17 @@ public function setDefaultFormat($format)
      */
     public function all()
     {
+        $regex = <<<REGEX
```

Comment:
We have few places with such long regular expressions, but you may want to take a look at how this is done in the `UrlValidator`: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/Constraints/UrlValidator.php#L23-L37